ofctrl/fgraphSwitch.go

```diff
@@ -89,16 +89,24 @@ func (self *OFSwitch) NewTable(tableId uint8) (*Table, error) {
 	return table, nil
 }
 
-// Delete a table.
+// Delete a table or specific flows.
 // Return an error if there are fgraph nodes pointing at it
-func (self *OFSwitch) DeleteTable(tableId uint8) error {
```

Comment:
Is this method useless? Maybe we could create a new one and keep this for later use.